client/client.go

```diff
@@ -172,11 +172,7 @@ func (r *NotaryRepository) Initialize(rootKeyID string) error {
 	err = r.tufRepo.InitRoot(false)
 	if err != nil {
 		logrus.Debug("Error on InitRoot: ", err.Error())
-		switch err.(type) {
-		case signed.ErrInsufficientSignatures, trustmanager.ErrPasswordInvalid:
-		default:
-			return err
-		}
+		return err
```

Comment:
This changes behaviour, I'm trying to remember why we permitted `ErrInsufficientSignatures` to continue (and later added `ErrPasswordInvalid`). Prior to the switch it was a very explicit type check. I'll have to dig back into the history and see if I can work out what the intent was.